tests/Tests/Isolated/Modules/FaxSMS/View/MessageUiProfileTest.php

```
@@ -0,0 +1,241 @@
+
+            // RingCentral sms exercises every table-backed tab in one profile.
+            $tabs = MessageUiProfile::tabs(ServiceType::RINGCENTRAL, 'sms');
+            foreach (['received', 'sent', 'messages', 'logs', 'alerts'] as $key) {
+                self::assertSame('table', $tabs[$key]['type']);
+                self::assertSame($expected[$key], $tabs[$key]['tableId']);
+            }
+        }
+
+        public function testReceivedRefreshesAndPanesCarryAnId(): void
+        {
+            foreach (self::tabKeyProvider() as [$service, $channel, $keys]) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$keys'. (undefined)

(UnusedLocalVariable)